### DISCUSSION TITLE
EC Probe Temperature Feed

### DISCUSSION THREAD

**Discussion by truvec** [tagyoureit/nodejs-poolController#895] [REM] [answered]
Before SWG setup(adding salt) EC probe read salt in the 600's and the pool store was in the same range. So I added the required salt to get to 3400. 

When I fired up the SWG this morning it read 3700. My probe is reading 3010@25c. 

Feeding the probe my water temp reads the salt as 3610@18.1 c. 

Should the EC probe be fed the water temp?

Thanks

**Comment by tagyoureit** [comment, marked as answer]
Yes, you should be feeding the temperature so the probe/chip can do the correct compensation.
